tests/backend_api/test_connector.py

```diff
@@ -123,23 +147,32 @@ def test_fetch_merged_items_mocked(
     )
 
 
-def test_get_merged_item_mocked(
+def test_fetch_all_merged_items_mocked(
     mocked_backend: MagicMock, merged_person: MergedPerson
 ) -> None:
-    mocked_return = {"items": [merged_person.model_dump()], "total": 1}
+    mocked_return = {"items": [merged_person.model_dump()], "total": 103}
     mocked_backend.return_value.json.return_value = mocked_return
 
     connector = BackendApiConnector.get()
-    response = connector.get_merged_item("NGwfzG8ROsrvIiQIVDVy")
+    items = list(
+        connector.fetch_all_merged_items(
+            query_string="Tintzmann",
+        )
+    )
 
-    assert response == merged_person
+    assert items == [merged_person] * 2  # should be two requests, one Tintzmann each
 
     assert mocked_backend.call_args == call(
```

Comment:
We expect 3 calls, don't we? One for getting the total and two for getting the items. Could you add an assert for making sure it's 3?